boot/LoadLinux.c

```diff
@@ -42,15 +42,15 @@ void try_elf_boot(char* data, int len);
 void BootPrintConfig(CONFIGENTRY *config) {
 	int CharsProcessed=0, CharsSinceNewline=0, Length=0;
 	char c;
-	printk("  Bootconfig : Kernel  %s \n", config->szKernel);
+	printk("  Bootconfig : Kernel  %s \n", config->opt.Linux.szKernel);
```

Comment:
Instead of all these `config->opt.Linux.` consider keeping a pointer / reference to this in something like `linux_config`; if this code path *only* ever needs Linux options, consider to only pass the linux struct type to this function (to make it impossible to accidentally read `config->opt.reactOS.szKernel` in the future).